app/views/ops/_settings_server_tab.html.haml

```diff
@@ -278,7 +278,7 @@
         = _("Security")
       .col-md-8
         = select_tag('webservices_security',
-                      options_for_select(%w(none ws-security), @edit[:new][:webservices][:security]),
+                      options_for_select([_("none"), _("ws-security")], @edit[:new][:webservices][:security]),
```

Comment:
Same as above: we'd be submitting / sending translated strings to the form controller.